Packs/CortexExposureManagement/Playbooks/Cortex_EM_-_Exposure_Issue.yml

```diff
@@ -0,0 +1,736 @@
+          "y": 1098
+        }
+      }
+  "15":
+    conditions:
+    - condition:
+      - - left:
+            iscontext: true
+            value:
+              simple: ${Core.CoreAsset.xdm__asset__tags}
+          operator: isNotEmpty
+          right:
+            value: {}
+      label: "yes"
+    continueonerrortype: ""
+    id: "15"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#default#':
+      - "4"
+      "yes":
+      - "14"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Checks if an asset has tags to extract indicators from.
+      id: fdffb487-b2b9-40a0-8af7-d57fafdeedf7
+      iscommand: false
+      name: Are there tags?
+      type: condition
+      version: -1
+    taskid: fdffb487-b2b9-40a0-8af7-d57fafdeedf7
+    timertriggers: []
+    type: condition
+    view: |-
+      {
+        "position": {
+          "x": 438,
+          "y": 924
+        }
+      }
+  "16":
+    conditions:
+    - condition:
+      - - left:
+            iscontext: true
+            value:
+              simple: ExtractedIndicators.Email
+          operator: isExists
+      label: "yes"
+    continueonerrortype: ""
+    id: "16"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#default#':
+      - "4"
+      "yes":
+      - "17"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Checks if emails were extracted from asset tags.
+      id: 7807bbb4-a494-4206-81d5-430d64b217a8
+      iscommand: false
+      name: Were emails found?
+      type: condition
+      version: -1
+    taskid: 7807bbb4-a494-4206-81d5-430d64b217a8
+    timertriggers: []
+    type: condition
+    view: |-
+      {
+        "position": {
+          "x": 438,
+          "y": 1247
+        }
+      }
+  "17":
+    continueonerrortype: ""
+    id: "17"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "4"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      keys:
+        simple: name,email,source,timestamp
+      val1:
+        simple: n/a
+      val2:
+        simple: ${ExtractedIndicators.Email}
+      val3:
+        simple: ${Core.CoreAsset.xdm__asset__provider}
+      val4:
+        simple: TIMESTAMP
+      context_key:
+        simple: vmassetownerunrankedraw
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: Builtin
+      description: |-
+        Automation used to more easily populate XSOAR context.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Instead of a value you can enter `TIMESTAMP` to get the current timestamp in ISO format. For example:
```

Comment:
This script simplifies how you add data to Cortex XSOAR's context. Use it to set static values or to map different values to existing context paths. Instead of a value you can enter TIMESTAMP to get the current timestamp in ISO format. For example:

Reply:
fixed